02_LabJat/bubble_sort.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func bubbleSort(input []int) []int {
+
```

Comment:
Remove blank lines at beginning and end of block

Reply:
Removed Blanks